app/frontend/src/cljs/teet/meeting/meeting_view.cljs

```diff
@@ -892,7 +892,7 @@
         location]
        [(tr "PDF")
         [:a {:href (common-controller/query-url :meeting/download-pdf
-                                                {:db/id (:db/id meeting)})
+                                                {:db/id (:db/id meeting) :meeting meeting})
```

Comment:
Shouldn't pass whole meeting as URL parameters, the payload may be large and the generated URL may be too long. There's already the meeting id, fetch the info on the backend instead of passing whole meeting in URL.